package.json

```diff
@@ -16,9 +16,22 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@types/bootstrap": "^5.1.6",
     "@types/express": "^4.17.13",
     "@types/fhir": "0.0.34",
+    "bootstrap": "^5.1.3",
     "concurrently": "^6.2.2",
-    "fhir-kit-client": "^1.7.2"
+    "fhir-kit-client": "^1.7.2",
+    "react-art": "^17.0.2",
+    "react-bootstrap": "^2.0.2",
+    "react-dom": "^17.0.2",
+    "react-modal": "^3.14.3",
+    "react-modal-bootstrap": "^1.1.1",
+    "react-native": "^0.66.2",
+    "react-native-cli": "^2.0.1",
+    "react-native-modal": "^13.0.0",
+    "react-native-modals": "^0.22.3",
+    "react-native-web": "^0.17.5",
+    "react-router-native": "^6.0.0"
```

Comment:
@liangfe5 

If we are not using these dependencies, could you please remove them? If we need them, please move them into package.json and package-lock.json in the client folder.